src/db.ts

```diff
@@ -35,7 +43,7 @@ const initializeDB = (async () => {
 
   PRIMARY KEY (id)
  )`);
-  await db.query(`CREATE TABLE IF NOT EXISTS answers (
+  await db.execute(`CREATE TABLE IF NOT EXISTS answers (
```

Comment:
It's not really relevant for these create table statements but I'm switching to execute because it gives access to [prepared statements](https://github.com/sidorares/node-mysql2#using-prepared-statements) which can improve sql performance